develop/tutorials/articles/194-adaptive-media/00-adaptive-media-images-rest-apis-intro.markdown

```diff
@@ -1,312 +1,365 @@
+
+    GET [server address]/o/adaptive-media-rest/images/content/[fileVersionId]/config/[configId]
+
+Now that you know how to use the */config* endpoint, you can learn about the
+*/data* endpoint next.
+
+### Data Endpoint [](id=data-endpoint)
+
+If you don’t know the config ID, or you don’t want to specify it, you can use
+the */data* endpoint. This endpoint receives a query array param
+(*query=parameter:value*) that specifies the properties of the image you want
+(e.g. width near 200px and height near 500px).
+
+**Status Codes**
+
+200<br />
+403 (user without permission)<br />
+400 (wrong or undefined query)<br />
+404 (fileVersion, fileEntry or no data found)
+
+**Query params**
+
+`original:boolean`: If the value is `true`, the original image is used as
+fallback.
+
+`string`: Used to pass properties of the image in the form of *property:value*.
+Queries can be passed for each image property.
+
+For example, if you want to retrieve an Adaptive Media version of the image
+close to 200px width, using 500px height to resolve ambiguities, you would use
+the following list of query params:
+
+    ?query=width:200&query=height:500
+
+Below is an example request pattern using the `fileEntryId`:
+
+    GET [server address]/o/adaptive-media-rest/images/content/file/[fileEntryId]/version/[version]/data?query=width:200&query=height:500
+
+Here is a request pattern that uses the `fileVersionId`:
+
+    GET [server address]/o/adaptive-media-rest/images/content/[fileVersionId]/data?query=width:200&query=height:500
+
+Here is a request pattern that adds the *original* query to the previous request:
+
+    GET [server address]/o/adaptive-media-rest/images/content/[fileVersionId]/data?query=width:200&query=height:500&query=original:true
+
+You can learn how to use the */variants* endpoint next.
+
+### Variants Endpoint [](id=variants-endpoint)
+
+The */variants* endpoint lets you retrieve the list of Adaptive Medias created
+from a file version, with all its metadata in a JSON format. In addition to the
+*query* param, and *original* param, you also have an *order* param
+(opposite to query, so you can't use both at the same time) that
+allows you to perform a strict order of the data: *order=property:true|false*
+(ascending or descending).
+
+**Status Codes**
+
+200<br />
+403 (user without permission)<br />
+400 (wrong or undefined query, wrong or undefined order or both provided)<br />
+404 (fileVersion, fileEntry not found)
+
+**Query params**
+
+`original:boolean`: If the value is `true`, the original image is used as
+fallback.
+
+`query`, `string`: Used to pass properties of the image in the form of
+*query=property:value*. Queries can be passed for each image property.
+
+`order`, `string:boolean`: Sets the order of the requested data in the form of
+*order=property:true|false* (`true` for ascending, or `false` for descending).
+This parameter can be passed for each image property.
+<!-- Is that correct? -->
```

Comment:
I have been talking to Sergio and we are going to remove this functionality (having more than one order) since it doesn't work anymore with the new configurations. The sintaxis will be the same (order=property:true|false) but will only accept one order attribute.